src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1600,14 +1648,45 @@ void NetworkOPsImp::pubServer ()
     if (!mSubServer.empty ())
     {
         Json::Value jvObj (Json::objectValue);
-        auto const& feeTrack = app_.getFeeTrack();
 
-        jvObj [jss::type]          = "serverStatus";
+        ServerFeeSummary f{app_.openLedger().current()->fees().base,
+            app_.getTxQ().getMetrics(*app_.openLedger().current()),
+            app_.getFeeTrack()};
+
+        // Need to cap to uint64 to uint32 due to JSON limitations
+        auto clamp = [](std::uint64_t v)
+        {
+            constexpr std::uint64_t max32 =
+                std::numeric_limits<std::uint32_t>::max();
+
+            return static_cast<std::uint32_t>(std::min(max32, v));
+        };
+
+
+        jvObj [jss::type] = "serverStatus";
```

Comment:
The lower camel case seems to be the convention for subscriptions, see https://ripple.com/build/rippled-apis/#subscribe